Source/WebCore/dom/ScriptElement.h

```diff
@@ -163,6 +163,7 @@ class ScriptElement {
 
     // https://w3c.github.io/trusted-types/dist/spec/#slots-with-trusted-values
     String m_trustedScriptText { emptyString() };
+    bool m_setFromDOMString : 1 { false };
```

Comment:
Please move this up so that they're next to other boolean member variables for better packing.

Reply:
Done!